lib/publiccloud/instance.pm

```diff
@@ -101,6 +102,8 @@ With C<<ssh_opts=>'...'>> you can overwrite all default ops which are in
 C<<$instance->ssh_opts>>.
 Use argument C<username> to specify a different username then
 C<<$instance->username()>>.
+
+This function is deprecated. Please use ssh_script_retry instead.
```

Comment:
I would also add `record_info` into function call with same message

Reply:
You mean to add a "warning" to the test run, when a deprecated function is used?

Yes makes sense, but I would do this as a second step, not within this PR. I would kickstart it here, and then replace them step by step to limit the possible fallout.

Reply:
yeah sure this will work to , but I actually don't any harm from adding this info box already with this PR .